client/src/pages/Results/GeneResult.tsx

```diff
@@ -0,0 +1,368 @@
+  const evidenceLevelOptions = Array.from(
+    new Set(results[activeTab].map((r) => r.evidence_level).filter(Boolean)),
+  )
+  const significanceOptions = Array.from(
+    new Set(results[activeTab].map((r) => r.significance).filter(Boolean)),
+  )
+
+  const sourceOptions = Array.from(
+    new Set(results[activeTab].flatMap((r) => r.sources).filter(Boolean)),
+  )
+
+  const clearAllFilters = () => {
+    setSelectedVariants([])
+    setSelectedDiseases([])
+    setSelectedTherapies([])
+    setSelectedEvidenceLevels([])
+    setSelectedSignificance([])
+    setSelectedSources([])
+  }
+
+  const activeFilters = [
+    ...selectedVariants.map((v) => ({ type: 'variant', value: v })),
+    ...selectedDiseases.map((d) => ({ type: 'disease', value: d })),
+    ...selectedTherapies.map((t) => ({ type: 'therapy', value: t })),
+    ...selectedEvidenceLevels.map((e) => ({ type: 'evidence_level', value: e })),
+    ...selectedSignificance.map((s) => ({ type: 'significance', value: s })),
+    ...selectedSources.map((src) => ({ type: 'source', value: src })),
+  ]
+
+  const removeFilter = (filter: { type: string; value: string }) => {
+    switch (filter.type) {
+      case 'variant':
+        setSelectedVariants((prev) => prev.filter((v) => v !== filter.value))
+        break
+      case 'disease':
+        setSelectedDiseases((prev) => prev.filter((d) => d !== filter.value))
+        break
+      case 'therapy':
+        setSelectedTherapies((prev) => prev.filter((t) => t !== filter.value))
+        break
+      case 'evidence_level':
+        setSelectedEvidenceLevels((prev) => prev.filter((e) => e !== filter.value))
+        break
+      case 'significance':
+        setSelectedSignificance((prev) => prev.filter((s) => s !== filter.value))
+        break
+      case 'source':
+        setSelectedSources((prev) => prev.filter((s) => s !== filter.value))
+        break
+    }
+  }
+
+  return (
+    <>
+      <Header />
+      <Box id="result-page-container" m={5}>
+        {loading && <CircularProgress />}
+        {error && <Alert severity="error">{error}</Alert>}
+        {!loading && (
+          <Box>
+            <Box
+              id="results-info-container"
+              sx={{ backgroundColor: 'white', padding: 5, borderRadius: 2 }}
+            >
+              <Typography variant="h4" mb={2} fontWeight="bold">
+                {searchQuery}
+              </Typography>
+              <Typography variant="h6" mb={2} fontWeight="bold" color="darkgrey">
+                Aliases: {aliases.join(', ')}
+              </Typography>
+              <Typography variant="body1" mb={2}>
+                {description}
+              </Typography>
+            </Box>
+            <Box
+              id="results-table-container"
+              sx={{ backgroundColor: 'white', padding: 5, borderRadius: 2, marginTop: 2 }}
+            >
+              <Tabs
+                onChange={(_, value) => setActiveTab(value)}
+                value={activeTab}
```

Comment:
I think we should (either with a mapping or just manually) make this value be capitalized in line with the context, so that it reads eg "Diagnostic Search Results" instead of "diagnostic Search Results"
<img width="338" height="103" alt="Screenshot 2025-10-06 at 4 10 53 PM" src="https://github.com/user-attachments/assets/03c567c8-3d08-4d47-a8cc-bc1f895313ae" />